include/xrpl/protocol/ErrorCodes.h

```diff
@@ -228,26 +228,17 @@ get_error_info(error_code_i code);
 
 /** Add or update the json update to reflect the error code. */
 /** @{ */
-template <class JsonValue>
-void
-inject_error(error_code_i code, JsonValue& json)
+inline void
```

Comment:
nit: This does not actually have to be in the header anymore. I'd move it to the cpp file